addons/html_builder/static/src/snippets/add_snippet_dialog.js

```diff
@@ -66,7 +66,7 @@ export class AddSnippetDialog extends Component {
         });
 
         onWillDestroy(() => {
```

Comment:
if root is created in onMounted, it should probably be destroyed in onWillUnmount